### PR TITLE
Add S3 logging configuration

### PR DESCRIPTION
**Pull request by guangie88**

### CODE REVIEW

File: docker-compose.emr.yml [hunk 1]
```diff
@@ -22,6 +22,11 @@ services:
     environment:
       AIRFLOW__CORE__FERNET_KEY: ZZvP_7JEhKfQ7v5sg1DV1o4b1DcGv3Tu9vJXzEKkxbE=
       AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql://fixme:fixme@localhost:5999/airflow
+      # FOR S3 LOGGING, COMMENT OUT THE BELOW 3 LINES TO DISABLE S3 LOGGING
+      # S3_LOG_FOLDER FORMAT: s3://bucket/[directories]
+      AIRFLOW__CORE__TASK_LOG_READER: s3.task
+      AIRFLOW__CORE__LOGGING_CONFIG_CLASS: log_config.LOGGING_CONFIG
```
Comment (chrissng) [inline docker-compose.emr.yml]: Should reference s3_log_config.LOGGING_CONFIG instead

File: README.md [hunk 2]
```diff
@@ -67,6 +67,16 @@ To start the Airflow containers in the master instance,
 sudo /usr/local/bin/docker-compose -f docker-compose.emr.yml up -d
 ```
 
+Logging to S3 bucket is enabled by the following lines in [`docker-compose.emr.yml`](./docker-compose.emr.yml):
+
+```yml
+AIRFLOW__CORE__TASK_LOG_READER: s3.task
+AIRFLOW__CORE__LOGGING_CONFIG_CLASS: log_config.LOGGING_CONFIG
```
Comment (chrissng) [inline README.md]: Same as below

**Review by chrissng — CHANGES_REQUESTED** [review]
I think there's a mistake in the environment variable

**Comment by guangie88** [comment]
Okay fixed, thanks for spotting that!

**Review by chrissng — APPROVED** [review]
LGTM

**Pull request merged**